Makefile

```diff
@@ -12,11 +12,15 @@ functional-test:
 	cram tests/*.t
 
 get-deps:
-	go get ./...
+	go get -t ./...
```

Comment:
Apparently `go get` ignores test files without the `-t` option. Not sure why. I guess it would be helpful if you used `go get` to install random binaries. Ugh.